packages/react-router/modules/Router.js

```diff
@@ -21,6 +22,11 @@ class Router extends React.Component {
 
   render() {
     const { children } = this.props
+    invariant(
+      children == undefined || React.Children.count(children) === 1,
+      'A <Router> must have at most one child element.'
```

Comment:
The phrase `must have at most` sounds odd to me. I think that `A <Router> can only have one child element.` would be clearer.